### PR TITLE
Actually make user-email page work

### PR DESCRIPTION
**Pull request by chrisnojima**
I had to fixup a lot of the view hierarchy all the way to nav since its incorrect. see comments in the code

@keybase/react-hackers

### CODE REVIEW

File: shared/common-adapters/user-card.native.js [hunk 1]
```diff
@@ -12,10 +12,12 @@ export default class UserCard extends Component<void, Props, void> {
     const url = this.props.username ? `https://keybase.io/${this.props.username}` : null
     return (
       <Box style={{...styles.container, ...this.props.outerStyle}}>
+        <Box style={styles.avatar}>
```
Comment (chrisnojima) [inline shared/common-adapters/user-card.native.js]: simpler to just wrap this in a box vs using absolute positioning. just leveraging flexbox seems to produce stabler results

File: shared/local-debug.native.js [hunk 2]
```diff
@@ -57,7 +57,7 @@ export function initTabbedRouterState (state) {
     ...state,
     tabs: {
       ...state.tabs,
-      [Tabs.loginTab]: createRouterState(['signup'], []),
+      [Tabs.loginTab]: createRouterState([], []),
```
Comment (chrisnojima) [inline shared/local-debug.native.js]: reset this

File: shared/login/forms/container.native.js [hunk 3]
```diff
@@ -18,21 +18,17 @@ export default ({children, onBack, style, outerStyle}: Props) => {
...
   container: {
     ...globalStyles.flexBoxColumn,
-    bottom: 0,
-    left: 0,
     padding: 16,
-    position: 'absolute',
-    flex: 1,
-    right: 0,
-    top: 6
+    flex: 1
```
Comment (chrisnojima) [inline shared/login/forms/container.native.js]: avoid using absolute unless we really need it

File: shared/login/signup/username-email-form.js [hunk 4]
```diff
@@ -6,14 +6,33 @@ import {bindActionCreators} from 'redux'
...
+  email: string
+}
 
 class UsernameEmailForm extends Component {
+  state: State;
+
+  constructor (props: Props) {
+    super(props)
+
+    this.state = {
```
Comment (chrisnojima) [inline shared/login/signup/username-email-form.js]: hoist this up and keep it as state so we don't need to use refs anymore

File: shared/nav.ios.js [hunk 5]
```diff
@@ -92,8 +92,13 @@ class Nav extends Component {
   }
 
   _renderContent (tab, module) {
+    const tabStyle = {
```
Comment (chrisnojima) [inline shared/nav.ios.js]: things that don't use the tabs go fullscreen

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by cjb** [comment]
:thumbsup:

**Pull request merged**